loama/src/components/LoginForm.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <header>
+    <h1>{{ title }}</h1>
+    <p class="tagline" v-if="subtitle">{{ subtitle }}</p>
+  </header>
+  <form @submit.prevent="login">
+    <fieldset>
+      <legend>
+        Solid Pod URL
+        <PhQuestion :size="24" @mouseover="showPopup = true;" @mouseleave="showPopup = false" />
+      </legend>
+      <label for="solid-pod-url">
+        <PhLink :size="24" class="icon" />
+        <input type="url" id="solid-pod-url" v-model="solidPodUrl" :placeholder="defaultSolidPodUrl" />
+      </label>
+      <p v-if="showWarning" class="warning">Invalid Solid Pod URL. Please check and try again.</p>
+    </fieldset>
+    <fieldset>
+      <LoButton @click.prevent="noPod" :disabled="isLoading" class="secondary" :left-icon="PhQuestion">
```

Comment:
So, for triggering the secondary variant of the button, it's `:varient="secondary"`
You can change this or keep this, just FYI

```suggestion
      <LoButton @click.prevent="noPod" :disabled="isLoading" :variant="secondary" :left-icon="PhQuestion">
```